scss/jquery/overrides/_ui-dialog.scss

```diff
@@ -0,0 +1,238 @@
+@import 'SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/modals';
+@import 'SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/style/modals';
+
+.crm-container.ui-dialog {
```

Comment:
there should be no reference of `.crm-container` in this partial

Reply:
As discussed, moved the content of `.ui-dialog` to a new mixin under `jquery/mixins/_ui-dialog.scss`. And reused it in `jquery/overrides/_ui-dialog.scss` and `civicrm/common/_modals.scss`.